test/HttpClient/Guzzle6HttpClientTest.php

```diff
@@ -28,6 +28,7 @@ protected function setUp()
 
         $this->mock = new MockHandler();
         $this->guzzle = new Client(['handler' => HandlerStack::create($this->mock)]);
+        $this->client = new Guzzle6HttpClient($this->guzzle);
```

Comment:
Could you revert these changes? Don't disagree with them, but they're unrelated.